reactioncommerce/reaction
# Fix checkout login permissions

This resolves #2657 

### Steps to test
1. Checkout without logged in and click "Continue as Guest" in first step. The "Welcome, it's so good to see you." message will be displayed and go to 2nd step.(checkoutLoginCompleted returned true)
1. Checkout and Signin in the first step. The "Welcome, it's so good to see you." message will be displayed and go to 2nd step.(checkoutLoginCompleted returned true).
